src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -697,6 +699,61 @@ private void UpdateTopologyCallback(object _)
             }));
         }
 
+        protected async Task SingleTopologyUpdateAsync(string[] initialUrls, Guid? applicationIdentifier = null)
+        {
+            if(Disposed)
+                return;
+
+            //fetch tag for each of the urls
+            Topology topology = new Topology() {Nodes = new List<ServerNode>(), Etag = TopologyEtag};
+            
+            foreach (var url in initialUrls)
+            {
+                var serverNode = new ServerNode
+                {
+                    Url = url,
+                    Database = _databaseName
+                };
+
+                try
+                {
+                    using (ContextPool.AllocateOperationContext(out JsonOperationContext context))
+                    {
+                        var command = new GetClusterTopologyCommand("single-topology-update");
+                        await ExecuteAsync(serverNode, null, context, command, shouldRetry: false, sessionInfo: null, token: CancellationToken.None)
+                            .ConfigureAwait(false);
+                        serverNode.ClusterTag = command.Result.NodeTag;
+                        serverNode.ServerRole = ServerNode.Role.Member;
+                    }
+                }
+                catch (AuthorizationException)
+                {
+                    // auth exceptions will always happen, on all nodes
+                    // so errors immediately
+                    _lastKnownUrls = initialUrls;
+                    throw;
+                }
+                catch (DatabaseDoesNotExistException)
+                {
+                    // Will happen on all node in the cluster,
+                    // so errors immediately
+                    _lastKnownUrls = initialUrls;
+                    throw;
+                }
+                catch (Exception e)
+                {
+                    serverNode.ClusterTag = "!";
+                    Logger.Info($"Error occurred while attempting to fetch the Cluster Tag for {url} in {nameof(SingleTopologyUpdateAsync)}", e);
```

Comment:
IsInfoEnabled (fix the other place as well)

Reply:
other place?